presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private boolean assumeCanonicalPartitionKeys;
 
     private boolean useOrcColumnNames;
+    private boolean orcBloomFiltersEnabled = true;
```

Comment:
I'm not sure we want to enable this by default for now because `https://issues.apache.org/jira/browse/ORC-101` makes the bloom filters on VARCHAR unreliable.   Once they patch the writer we could enable it by default but skip the VARCHAR columns for older files.  Unfortunately there isn't a good answer here.